go.mod

```diff
@@ -24,21 +24,21 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.5.0
+	github.com/stretchr/testify v1.6.1
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
 	go.opencensus.io v0.22.3
 	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
+	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e
 	golang.org/x/tools v0.0.0-20210106214847-113979e3529a
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013
-	google.golang.org/grpc v1.27.0
+	google.golang.org/grpc v1.27.1
```

Comment:
**thought (non-blocking)**: We really should upgrade the SDK grpc versions 😄

Reply:
I created https://github.com/googleforgames/agones/issues/2196 for the golang SDK. We should create ones for the others that need it (e.g. rust was just updated, so probably doesn't).